micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -2,66 +1,65 @@
 import { Button } from '@egovernments/digit-ui-components';
+import _ from "lodash";
 
-const MDMSView = ({...props}) => {
+const MDMSView = ({ ...props }) => {
   const history = useHistory()
   const { t } = useTranslation()
   const [showToast, setShowToast] = useState(false);
-  let { moduleName, masterName, tenantId,uniqueIdentifier } = Digit.Hooks.useQueryParams();
-  let {from, screen, action} = Digit.Hooks.useQueryParams()
+  let { moduleName, masterName, tenantId, uniqueIdentifier } = Digit.Hooks.useQueryParams();
+  let { from, screen, action } = Digit.Hooks.useQueryParams()
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'from' is never reassigned.

'screen' is never reassigned.

'action' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)